addons/point_of_sale/models/pos_config.py

```diff
@@ -307,6 +309,11 @@ def _check_session_state(self):
         if open_session:
             raise ValidationError(_("You are not allowed to change the cash control status while a session is already opened."))
 
+    @api.constrains('rounding_method')
+    def _check_rounding_method_strategy(self):
```

Comment:
you have to add a constraint on the roundig method (an overwrite in pos) because here you can modify the rounding method once it is set on the config.

Reply:
sorry it is below forget about it